src/components/social-link-button/SocialLinkButton.tsx

```diff
@@ -0,0 +1,11 @@
+import classNames from "./SocialLinkButton.module.scss";
+
+function SocialLinkButton({ socialLink }: { socialLink: string }) {
```

Comment:
you can also use arrow functions for it and extract types to interface so it would look like this:
interface Props {
    socialLink: string
}

const SocialLinkButton = ({ socialLink }: Props) => {

Reply:
with more props such solution will be way more readable